src/App.jsx

```diff
@@ -12,6 +13,6 @@ export const App = () => (
   <div className="App">
     <h1>React Article</h1>
 
-    {/* Add an <Article /> and pass `title`, `text` and `date` from the `article` object as props */}
+    <Article title={article.title} date={article.date} text={article.text} />
```

Comment:
[CODE STYLE] - If you have < 3 attribues on a tag and the values are short write the tag in one line (to make it easier to write and read).
```suggestion
    <Article 
      title={article.title} 
      date={article.date} 
      text={article.text}
    />
```

Reply:
Agree. Need to fix